docs/nx-cloud/concepts/cache-security.md

```diff
@@ -0,0 +1,59 @@
+# Remote Cache Security
+
+{% callout type="warning" title="Use Caution With Read-Write Tokens" %}
+Read-write tokens allow full write access to your remote cache. They should only be used in trusted environments. For instance, open source projects should only use read-write tokens as secrets configured for protected branches (e.g, main). Read-only tokens should be used in all other cases.
+{% /callout %}
+
+A cache allows you to reuse work that has already been done, but it also introduces a potential security risk - cache poisoning. A poisoned cache is one where the cached files have been altered in some way by a malicious actor. When a developer or the CI pipeline use that poisoned cache, the task output will be what the malicious actor wants instead of the correct task output.
+
+Nx takes security seriously and has put in place many precautions (we're [SOC 2 compliant](https://security.nx.app)). Listed below are some precautions that you need to take in your own codebase.
+
+## Recommended Precautions
+
+In order to keep your cache secure, there are a few steps we recommend you take:
+
+- Specify a [read-only token](/nx-cloud/recipes/security/access-tokens) in the `nx.json` file
```

Comment:
can we add a little more details to these recommendations? Like why a read-only why skip the cache when doing releases/deployments etc.. Maybe we can have it like

- **read-only token in `nx.json`** - {explainer coming here...}
- ...

Or have it in table form?